web-app/src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -52,6 +54,13 @@ const BookingModalForm = props => {
     }
   };
 
+  const remainingDays = (getTotalDays - getApprovedDays());
```

Comment:
Should `getTotalDays` be called here rather than referenced? I.e. `getTotalDays() - getApprovedDays()`